libraries/consul_service.rb

```diff
@@ -55,6 +55,7 @@ def default_environment
         {
           'GOMAXPROCS' => [node['cpu']['total'], 2].max.to_s,
           'PATH' => '/usr/local/bin:/usr/bin:/bin',
+          'SHELL' => node['consul']['service_shell'],
```

Comment:
Hm... I'd vote for `node['consul']['service_shell']` because the name of this argument os exactly what we expect here. But let's ask @johnbellone too. 

But in any case, the element should be appended using `if` condition, since the attribute is not defined by default. For example:
```
def default_environment
  env = {
    'GOMAXPROCS' => [node['cpu']['total'], 2].max.to_s,
    'PATH' => '/usr/local/bin:/usr/bin:/bin',
  }
  env['SHELL'] = node['consul']['service_shell'] if node['consul']['service_shell']
  env
end
```